docs/appendix/apis/components/arm.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Arm API"
+linkTitle: "Arm"
+weight: 20
+type: "docs"
+description: "The arm API allows you to give commands to your arm components for linear motion planning, including self-collision prevention and obstacle avoidance."
```

Comment:
The first half of each component description is redundant with the "<component> API" title. Can be much less verbose.
```suggestion
description: "Give commands to your arm components for linear motion planning, including self-collision prevention and obstacle avoidance."
```
or better yet
```suggestion
description: "Commands for linear motion planning, including self-collision prevention and obstacle avoidance."
```